pom.xml

```diff
@@ -272,9 +272,9 @@
 		<spring-cloud-stream.version>Elmhurst.BUILD-SNAPSHOT</spring-cloud-stream.version>
 		<spring-cloud-netflix.version>2.0.0.BUILD-SNAPSHOT</spring-cloud-netflix.version>
 		<spring-cloud-openfeign.version>2.0.0.BUILD-SNAPSHOT</spring-cloud-openfeign.version>
-		<brave.version>4.18.2</brave.version>
+		<brave.version>4.19.0</brave.version>
 		<!-- Version set until zipkin-junit gets defined in Brave BOM -->
```

Comment:
this comment lies a little even if once true.. it is actually about zipkin's test jar I think.